models/schemas/userSchema.ts

```diff
@@ -79,7 +79,7 @@ const UserSchema = new mongoose.Schema<User>({
     select: false,
     required: true,
     minlength: 8,
-    maxlength: 50,
+    maxlength: 64,
```

Comment:
need to increase this so we can use insert()

Reply:
Is this because of using
`password: await bcrypt.hash('test1234', PASSWORD_SALTROUNDS),`
in initializeLocalDb?

What if we use `create` with an array instead of `insertMany`? That's how we do it in `api/signup` and it ends up calling the pre save hook in `UserSchema`